RubeesEat/wwwroot/css/UserHomePage.css

```diff
@@ -0,0 +1,118 @@
+.positive-balance {
+    color: green;
+}
+
+.negative-balance {
+    color: red;
+}
+
+#list {
+    list-style: none;
+    padding: 0;
+    margin: 10px;
+}
+
+#list li {
+    margin: 10px 10%;
+    padding: 10px;
+    background-color: #fff;
+    border-radius: 5px;
+    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1);
+    transition: all 0.3s ease;
+}
+
+#list li:hover {
+    transform: scale(1.02);
+    z-index: 1;
+}
+
+@media (max-width: 1024px) {
+    #balanceChanges li {
+        margin: 10px 5%;
+    }
+}
+
+@media (max-width: 768px) {
+    #balanceChanges li {
+        margin: 10px 2%;
+    }
+}
+
+
+#list li:hover {
+    background-color: #f2e2b2;
+}
+
+.clickBillDetails {
+    text-decoration: none;
+    color: #333;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 10px;
+    transition: background-color 0.3s ease;
+}
+
+#hello {
+    font-size: 3em;
+    margin-top: 30px;
+}
+
+#balance {
+    font-size: 2.5em;
+    text-align: center;
+}
+
+
+@media (max-width: 1024px) {
+    #hello {
+        font-size: 2.5em;
+    }
+
+    #balance {
+        font-size: 2em;
+    }
+}
+
+@media (max-width: 768px) {
+    #hello {
+        font-size: 2em;
+    }
+
+    #balance {
+        font-size: 1.75em;
+    }
+}
+
+@media (max-width: 480px) {
+    #hello {
+        font-size: 1.5em;
+    }
+
+    #balance {
+        font-size: 1.5em;
+    }
+}
```

Comment:
I would put the @media selectors together. As far as I know it is better to put them on top, to ensure mobile-first, but I couldn't find any info on that online, so you can disregard that if you want